### ISSUE TITLE
Update Chapter 20: Access to Data

### ISSUE THREAD

**Issue by claudio-rosati** [ControlSystemStudio/org.csstudio.docbook#10]
Hello all,

I think _Chapter 20. Access to Data_ should be updated, specially after the integration of the new DIIRT library by @shroffk  and @kasemir.

I'll add a new paragraph related to the initialization of the `diirt.home` preference inside `plugin_customization.ini` where `@user.home` or `platform:` syntax can be used. Or do you think it is better to add it in _Chapter 6. Hierarchical Preferences_?

**Comment by kasemir** [comment]
Yes, when we're done, we need to update the access to data chapter.

When you're done with the preference changes, I agree that they should be mentioned in the Hierarchical Preferences chapter.
